x-pack/plugins/security_solution/public/flyout/left/components/correlations_cases_table.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { type EuiBasicTableColumn, EuiInMemoryTable } from '@elastic/eui';
+import type { RelatedCaseInfo } from '@kbn/cases-plugin/common/api';
+import React, { type FC } from 'react';
+import { CaseDetailsLink } from '../../../common/components/links';
+
+import * as i18n from './translations';
+
+export interface CorrelationsCasesTableProps {
+  cases: RelatedCaseInfo[];
+}
+
+const columns: Array<EuiBasicTableColumn<RelatedCaseInfo>> = [
+  {
+    field: 'title',
+    name: i18n.CORRELATIONS_CASE_NAME_COLUMN_TITLE,
+    truncateText: true,
+    render: (value: string, caseData: RelatedCaseInfo) => (
+      <CaseDetailsLink detailName={caseData.id} title={caseData.title}>
+        {caseData.title}
+      </CaseDetailsLink>
+    ),
+  },
+  {
+    field: 'status',
+    name: i18n.CORRELATIONS_CASE_STATUS_COLUMN_TITLE,
```

Comment:
this guy isn't defined in the `translations.ts` file, which is why there isn't any header for the status column in the UI right now
<img width="764" alt="Screenshot 2023-06-12 at 3 52 42 PM" src="https://github.com/elastic/kibana/assets/17276605/7ddd97fd-5b71-4797-badf-8e9f44ffe3c8">

Reply:
thanks, good catch!